test/videojs-contrib-hls.test.js

```diff
@@ -2072,6 +2061,54 @@ QUnit.test('resolves relative key URLs against the playlist', function() {
               'resolves the key URL');
 });
 
+QUnit.test('adds 1 empty audio track if we have not parsed any, and the playlist is loaded', function() {
+  this.player.src({
+    src: 'manifest/master.m3u8',
+    type: 'application/vnd.apple.mpegurl'
+  });
+
+  QUnit.equal(this.player.audioTracks().length, 0, `zero audio tracks at load time`);
+
+  openMediaSource(this.player, this.clock);
+
+  // master
+  standardXHRResponse(this.requests.shift());
+
+  QUnit.equal(this.player.audioTracks().length, 1, `one audio track after load`);
+});
+
+QUnit.test('adds audio tracks if we have parsed some from a playlist', function() {
+  this.player.src({
+    src: 'manifest/multipleAudioGroups.m3u8',
+    type: 'application/vnd.apple.mpegurl'
+  });
+
+  QUnit.equal(this.player.audioTracks().length, 0, `zero audio tracks at load time`);
+
+  openMediaSource(this.player, this.clock);
+
+  // master
+  standardXHRResponse(this.requests.shift());
+  let at = this.player.audioTracks();
+
+  QUnit.equal(at.length, 3, `three audio tracks after load`);
+
+  QUnit.equal(at[0].label, 'English', `track 1 - label = NAME`);
+  QUnit.equal(at[0].enabled, true, `track 1 - enabled = DEFAULT`);
+  QUnit.equal(at[0].language, 'eng', `track 1 - language = LANG`);
+  QUnit.equal(at[0].kind, 'main', `track 1 - kind = main if DEFAULT is YES`);
```

Comment:
Another non-critical comment: you can use `deepEqual` to cut down on the amount of typing you need to do for these sorts of tests.